src/oss/langchain-deploy.mdx

```diff
@@ -1,7 +1,164 @@
 ---
-title: (Coming Soon) Deploy
+title: Deploy
 ---
 
 import AlphaCallout from '/snippets/alpha-lc-callout.mdx';
 
-<AlphaCallout />
\ No newline at end of file
+<AlphaCallout />
+ 
+ LangGraph Platform is the easiest way to deploy agents. Traditional hosts are built for stateless, short-lived web apps, while LangGraph is **purpose-built for stateful, long-running agents**, so you can go from repo to reliable cloud deployment in minutes.
```

Comment:
don't love the "easiest" adjective -- it sounds like sales / marketing language, but also one that isn't inspiring since it doesn't explain what the  platform enables. i.e. ,"easy" is very vague